tests/lib/config/config-initializer.js

```diff
@@ -232,25 +232,25 @@ describe("configInitializer", function() {
                     format: "JSON",
                     commonjs: false
                 };
-                origLog = console.log;
-                console.log = function() {}; // necessary to replace, because of progress bar
+
+                sandbox = sinon.sandbox.create();
+                sandbox.stub(console, "log"); // necessary to replace, because of progress bar
+
                 process.chdir(fixtureDir);
+
                 try {
                     config = init.processAnswers(answers);
                     process.chdir(originalDir);
                 } catch (err) {
 
-                    // if processAnswers crashes, we need to be sure to restore cwd and console.log
-                    console.log = origLog;
+                    // if processAnswers crashes, we need to be sure to restore cwd
                     process.chdir(originalDir);
                     throw err;
+                } finally {
+                    sandbox.restore();  // restore console.log()
                 }
             });
 
-            beforeEach(function() {
-                console.log = origLog;
-            });
```

Comment:
I ran `npm test` and didn't see any strange output, is that sufficient? Or is there more I need to do to check this case out?